JavaScript/test/runner-number.js

```diff
@@ -10,7 +10,7 @@ var modelFunctions = {
     'PercentModel': (input, culture, options) => Recognizer.recognizePercentage(input, culture, options, false),
     // TODO: Implement number range model in javascript
     'NumberRangeModel': (input, culture, options) => null,
-    'CustomNumberModel': (input, culture, options) => getCustomNumberModel(culture).parse(input)
```

Comment:
If custom model was only used to test this particular case, I think we can remove it.
The tests for the Chinese Units model will already make use of it.